velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -301,10 +321,22 @@ VeloxToSubstraitExprConvertor::toSubstraitNullLiteral(
       substraitField->mutable_null()->set_allocated_fp64(nullValue);
       break;
     }
+    case velox::TypeKind::UNKNOWN: {
+      ::substrait::Type_UserDefined* nullValue =
+          google::protobuf::Arena::CreateMessage<::substrait::Type_UserDefined>(
+              &arena);
+      nullValue->set_nullability(
+          ::substrait::Type_Nullability_NULLABILITY_NULLABLE);
+      nullValue->set_type_reference(0);
```

Comment:
What does 0 mean here? Is it the type ID defined somewhere else?

Reply:
yes, it's the ID we defined `unknownType->set_type_anchor(0);`  in `VeloxToSubstraitPlan.cpp`